nncf/experimental/tensor/functions/torch_numeric.py

```diff
@@ -227,29 +227,30 @@ def _(
     a: torch.Tensor,
     q: Union[float, List[float]],
     axis: Optional[Union[int, Tuple[int]]] = None,
-    keepdims: Optional[bool] = None,
+    keepdims: Optional[bool] = False,
```

Comment:
This change is not aligned with https://github.com/openvinotoolkit/nncf/blob/99b5b85aaa5ee0c200edfa3de1ef8ff27a640385/nncf/experimental/tensor/functions/numeric.py#L454

Reply:
Changed to `keepdims: bool = False,` according https://numpy.org/doc/stable/reference/generated/numpy.quantile.html